libclamav/readdb.c

```diff
@@ -2943,6 +2943,30 @@ static int cli_loadcdb(FILE *fs, struct cl_engine *engine, unsigned int *signo,
     return CL_SUCCESS;
 }
 
+static cl_error_t setHash(const char *const token, uint8_t hashDest[SHA1_HASH_SIZE],
```

Comment:
`setHash` doesn't seem to match the naming convention in the rest of the file (i.e., camelCase vs. snake_case).  Is that intentional?

Reply:
A side curiosity -- what's the standard for the meaning of a "hash" within the ClamAV codebase?  Depending on context, it either means a dictionary or a cryptographic digest (or even just a `#`).  IMHO, it's usually a little clearer to use a term like "sha1", "sha2_256", "sha3_256", *etc.* for digests.

Reply:
True we generally do snake_case, not camelCase.  
It would also be helpful to give a descriptive function comment.

Reply:
I agree, I changed it to set_sha1.